### PR TITLE
[improvement](profile)add sum/avg rpc time 

### PR DESCRIPTION
**Pull request by Mryange**
## Proposed changes

beforce
image
after
image

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

File: be/src/pipeline/exec/exchange_sink_buffer.cpp [hunk 1]
```diff
@@ -305,25 +306,29 @@ bool ExchangeSinkBuffer::_is_receiver_eof(InstanceLoId id) {
...
-        iter++;
-    }
-    while (iter != _instance_to_rpc_time.end()) {
-        int64_t cur_val = iter->second;
-        local_max_time = cur_val > local_max_time ? cur_val : local_max_time;
-        local_min_time = cur_val < local_min_time ? cur_val : local_min_time;
-        iter++;
+    int64_t local_min_time = INT64_MAX;
+    for (auto& [id, time] : _instance_to_rpc_time) {
+        if (time == 0) {
```
Comment (HappenLee) [inline be/src/pipeline/exec/exchange_sink_buffer.cpp:311]: if (time != 0) {
        local_max_time = std::max(local_max_time, time);
        local_min_time = std::min(local_min_time, time);
}

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run p1

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**